resources/assets/uad_lists.json

```
@@ -17630,6 +17630,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.google.android.safetycenter.resources",
+    "list": "Google",
+    "description": "Related to Google Safety Center (https://safety.google). Probably effects malware detection in new app installs, gmail, and chrome. Use a libre spam-blocking and DNS-blocking solution instead.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
Yeah, maybe advanced is better, as it's the only (default) security suite that's built-in. Disabling increases the attack surface for a user.